web/multinode/src/app/router/index.ts

```diff
@@ -137,6 +137,11 @@ const allowedRoutesNames = [Config.AddFirstNode.name, Config.Welcome.name];
  * Redirect to Add first node screen if so.
  */
 router.beforeEach(async(to, _from, next) => {
+
+    if(to.path === '/') {
```

Comment:
there should be a space after `if`. That's why the build is failing due to lint error

Reply:
Done.